client/src/components/Map/components/SuperBlock.js

```diff
@@ -90,7 +94,49 @@ export class SuperBlock extends Component {
   }
 
   render() {
-    const { superBlock, isExpanded, toggleSuperBlock } = this.props;
+    const {
+      completedChallenges,
+      superBlock,
+      nodes,
+      isExpanded,
+      toggleSuperBlock
+    } = this.props;
+
+    // since the nodes have been filtered based on isHidden, any blocks whose
+    // nodes have been entirely removed will not appear in this array.
+    const blocksForSuperBlock = nodes.filter(
+      node => node.superBlock === superBlock
+    );
+
+    const certificationProjects = blocksForSuperBlock.filter(block =>
+      projectDashedBlockNameRE.test(block.block)
+    );
+
+    const completedProjectCount = certificationProjects.filter(block =>
+      completedChallenges.some(completedId => block.id === completedId)
+    ).length;
```

Comment:
```suggestion
    const completedProjectCount = certificationProjects.filter(({ id }) =>
      completedChallenges.includes(id)
    ).length;
```
`includes` is a little more descriptive and easier to read.

Reply:
I too felt that include would be better, but `some` is used here:

https://github.com/freeCodeCamp/freeCodeCamp/blob/aa1acbe68fe642ab49396c38e16e1c8446653147/client/src/components/Map/components/Block.js#L138-L140

Reply:
Should I change it to include?

Reply:
Yes please.  Could you add it as a separate `refactor: ` commit?

Reply:
Done! Can you take a look?

Reply:
Perfect, thank you!